R/tidyme.R

```diff
@@ -87,9 +87,16 @@ tidyme.survfit <- function(x, ...) {
       retme[["n.strata"]] <- rep(x[["n"]], x[["strata"]])
     }
   }
-  
+
   attr(retme, "survfit_object") <- survfit_object
+  # modify strata label, removing ref to raw variable name
+  if (!is.null(purrr::pluck(survfit_object, "estimate_KM_args", "strata"))) {
+    for (stratum in purrr::pluck(survfit_object, "estimate_KM_args", "strata")) {
+      retme[["strata"]] <- stringr::str_remove(retme[["strata"]], stringr::fixed(paste0(stratum, "=")))
```

Comment:
We decided not to depend on stringr. BASE R should be sufficient for sting manipulation.

Reply:
gone!